CONTRIBUTING.md

```diff
@@ -64,6 +64,25 @@ If the bug flew under the radar, and got discovered later:
 
 ## Quick start
 
+> [!NOTE] go version
+> We are very picky about the go version we use for development.
+> Because of some inconsistencies between go versions and golangci-lint and what we use in CI,
```

Comment:
```suggestion
> We are very picky about the go version we use for development,
> because of some inconsistencies between go, golangci-lint and what we use in CI.
```